hack/prom-rule-ci/rule-spec-dumper.go

```diff
@@ -24,32 +26,55 @@ func main() {
 		os.Exit(1)
 	}
 
-	targetFile := os.Args[1]
+	targetRules := os.Args[1]
+	targetFile := os.Args[2]
 
-	err := rules.SetupRules()
+	var promRule *promv1.PrometheusRule
+
+	if targetRules == "hyperconverged" {
+		promRule = buildHyperconvergedRule()
+	} else if targetRules == "observability" {
+		promRule = buildObservabilityRule()
+	} else {
+		fmt.Printf("ERROR: invalid target rules: %s\n", targetRules)
```

Comment:
Two comment here:
1. all the rest of the errors, cause `panic` and here we print error and exiting with error code. Personally, I prefer this approach over panic, but this is not consistent with the rest of the code. I think we should change all the rest to this pattern, or change this one to panic.
2. print to stdout:
```suggestion
		fmt.Fprintf(os.Stdout, "ERROR: invalid target rules: %s\n", targetRules)
```

Reply:
changed all the panics